torch_audiomentations/utils/multichannel.py

```diff
@@ -4,4 +4,4 @@ def is_multichannel(samples):
     :param samples:
     :return:
     """
-    return len(samples.shape) > 1
+    return len(samples.shape) > 2
```

Comment:
Just to be sure, this assumes that we expect the batch size to be there? 
Just thinking out loud but if the transforms are applied in the `Dataset`, they're applied to individual examples, without batch dim, right?

Reply:
It really depends. I would imagine applying augmentations should be done when iterating through a `DataLoader` just before it goes to the model for training, for example. At that time, a `torch.Tensor` input is already present and if the batch size is set to 1, the first dimension will be present.

I think even if it is a mono audio signal, the number of channels dimension should be set to 1 and can be removed by applying `squeeze` to the tensor.

Reply:
From what I've seen, the transforms are applied in the `__getitem__` of the `Dataset` (e.g. in albumentations or torchvision). This means that the transform is independent for every element in the batch, and that there is supposedly no batch size when applying the transform.
I think I would agree to have `(batch, 1, samples)` for mono audio because that's what is expected by Conv1D, and it solves one ambiguity. 
And then, we could have `is_multichannel` and `is_batched_multichannel`.

Reply:
For the sake of limiting the scope of this PR, I'd say we can assume 3 dimensions in the input/output. That way, processing single examples can be supported with batch size = 1. We should definitely keep options open for re-evaluating/changing this afterwards though.